test/run-tests.py

```diff
@@ -0,0 +1,115 @@
+import contextlib
+import logging
+import os
+import subprocess
+import sys
+import tempfile
+import time
+from enum import Enum
+
+EXPECTED_TIME_TO_BOOT = 60  # seconds
+RESET = "\033[0m"
+BOLD = "\033[1m"
+RED = "\033[31m"
+OBJECTS = os.environ.get("OBJECTS", tempfile.mkdtemp())
+OUTPUT_DIR = os.environ.get("OUTPUT_DIR", tempfile.mkdtemp())
+OSBUILD = os.environ.get("OSBUILD", "../osbuild")
```

Comment:
Please use a path relative to `__file__`, so that I can run `test/run-tests.py`.